setup.py

```diff
@@ -4,10 +4,23 @@
 from setuptools import setup
 from pkg_resources import parse_version
 import platform
+import sys
+import warnings
 import os
+import subprocess
 
 current_platform = platform.system().lower()
 extra_includes = []
+if current_platform == "darwin":
```

Comment:
If OS X and Linux have the same procedures, that would work. I have to test it on Linux still, so I will circle back to you on that!